XMonad/Doc/Extending.hs

```diff
@@ -154,23 +147,23 @@ In the @XMonad.Hooks@ namespace you can find modules exporting
 hooks. Hooks are actions that xmonad performs when certain events
 occur. The three most important hooks are:
 
-* 'XMonad.Core.manageHook': this hook is called when a new window
+* 'XMonad.Core.manageHook': this hook is called when a new window that
   xmonad must take care of is created. This is a very powerful hook,
   since it lets us examine the new window's properties and act
   accordingly. For instance, we can configure xmonad to put windows
   belonging to a given application in the float layer, not to manage
   dock applications, or open them in a given workspace. See
-  "XMonad.Doc.Extending#Editing_the_manage_hook" for more information on
-  customizing 'XMonad.Core.manageHook'.
+  "XMonad.Doc.Extending#Editing_the_manage_hook" for more information
+  on customizing 'XMonad.Core.manageHook'.
 
 * 'XMonad.Core.logHook': this hook is called when the stack of windows
```

Comment:
There is no `XMonad.Core`; wouldn't this be confusing?

Reply:
This is only broken on Hackage. When the docs are generated together in one `stack` invocation, like they're for https://xmonad.github.io/xmonad-docs/, this link works fine.